src/components/Link/Link.tsx

```diff
@@ -18,9 +19,12 @@ export default function Link({
   isJump = false,
   isJumpLeft = false,
   ...others
-}: LinkProperties & React.HTMLProps<HTMLAnchorElement>): JSXElement {
-  const cname = ['a-link', others.className];
-  if (type === 'list') cname.push('list-link');
+}: LinkProperties): JSXElement {
+  const cname = [others.className];
+
+  if (type === 'list') cname.push('m-list_link');
+  else cname.push('a-link');
```

Comment:
The multi-line conditional without curlies hurts my brain. How would you feel about:

```suggestion
  if (type === 'list') {
    cname.push('m-list_link');
  } else {
    cname.push('a-link');
  }
```

or if we want to keep everything a one-liner:

```js
if (type === 'list') cname.push('m-list_link');
if (type !== 'list') cname.push('a-link');
```